java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -1804,6 +1804,14 @@ public ColumnVector stringContains(Scalar compString) {
     }
   }
 
+  public ColumnVector matchesRe(String pattern) {
```

Comment:
Can we have some javadocs explaining this?  REs are complicated and just pattern is not much of an explanation.

Reply:
Sorry - missed that. Addressing in the next version.